service/src/test/java/bio/terra/pipelines/service/PipelinesServiceTest.java

```diff
@@ -272,7 +320,7 @@ void validateRequiredInputPresent(
       Boolean isRequired, Map<String, Object> inputs, Boolean shouldPassValidation) {
     PipelineInputDefinition inputDefinition =
         new PipelineInputDefinition(
-            1L, "input_name", PipelineInputTypesEnum.INTEGER.toString(), isRequired);
+            1L, "input_name", PipelineInputTypesEnum.INTEGER.toString(), isRequired, true, null);
```

Comment:
is there a reason we use `1L` here and `3L` in the TestUtils stuff?  do the numbers actually matter or are they just random

Reply:
here it's arbitrary because the validation doesn't care about pipelineId at this point. ok to just make a note of that here?

Reply:
yea thats fair!